devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -256,8 +257,7 @@ private Map getBatteryResult(rawValue) {
 				def pct = batteryMap[volts]
 				if (pct != null) {
 					result.value = pct
-                    def value = pct
```

Comment:
Sorry, @Bijnagte stopped by and explained it. The value isn't inserted here, it's inserted later using the value stored in the event.